junit/src/main/java/org/kaazing/k3po/junit/rules/SpecificationStatement.java

```diff
@@ -112,10 +122,11 @@ public void evaluate() throws Throwable {
                 }
             }
 
+            // Stefan - not sure what the comment bellow is about. There is no join() method in the statement or rule class.
```

Comment:
I think this is already there, in the assertTrue. What I don't understand is the join() method specified in the comments. There is no join() method in the statement or rule. Maybe it is an older comment and the join() method was removed ? Then we can remove or fix the comment.